File: OffByOne/src/OffByOne.LanguageCore/Ast/Expressions/Literals/BooleanLiteral.cs
namespace OffByOne.LanguageCore.Ast.Expressions.Literals
{
    public class BooleanLiteral : Expression
    {
        public BooleanLiteral(bool value)
        {
            this.Value = value;
        }
        public bool Value { get; set; }
    }
}

File: AlternativeQL/src/org/ql/ast/expression/BooleanLiteral.java
package org.ql.ast.expression;

import org.ql.ast.Expression;

public class BooleanLiteral implements Expression {
    private boolean booleanLiteral;

    public BooleanLiteral(boolean booleanLiteral) {
        this.booleanLiteral = booleanLiteral;
    }

    public boolean getBooleanLiteral() {
        return booleanLiteral;
    }
}
